vacuum-advisor/index.mdx

```diff
@@ -0,0 +1,10 @@
+---
+title: 'VACUUM Advisor'
+backlink_href: /docs
+backlink_title: 'Documentation'
+---
+
+The pganalyze VACUUM Advisor is a tool to help tuning VACUUM.
```

Comment:
```suggestion
The pganalyze VACUUM Advisor is a tool to help tune VACUUM.
```
or
```suggestion
The pganalyze VACUUM Advisor is a tool to help tune VACUUM settings.
```

Reply:
Note: this section will be overwritten by Lukas, it's more as a placeholder. I'll update though